website/src/content/docs/blog/biome-wins-prettier-challenge.mdx

```diff
@@ -0,0 +1,145 @@
+
+#### New formatting options
+
+With this challenge, we added new options to the formatter:
+- [`lineEnding`](/reference/configuration#formatterlineending)
+
+  Use this option to match the line endings of your OS. We support `lf` (line feed - `\n`), `cr` (carriage return - `\r`) and `crlf` (carriage return line feed - `\r\n`).
+
+- [`bracketSameLine`](/reference/configuration#formatterbracketsameline)
+
+  ```jsx title="example.js"
+  // Existing behavior. Now also the default, meaning `bracketSameLine: true`.
+  <Foo
+    className={somethingReallyLongThatForcesThisToWrap}
+    anotherReallyLongAttribute={withAValueThatsSurelyTooLong}
+    soThatEverythingWraps
+  >
+    Hello
+  </Foo>
+
+  <Foo
+    selfClosingTags={likeThisOne}
+    stillPlaceTheBracket={onItsOwnLine}
+    toIndicateThat={itClosesItself}
+  />
+  ```
+
+  After formatting with `"bracketSameLine": true`:
+
+  ```jsx title="example.js"
+  // New behavior, with `bracketSameLine: false`.
+  <Foo
+    className={somethingReallyLongThatForcesThisToWrap}
+    anotherReallyLongAttribute={withAValueThatsSurelyTooLong}
+    soThatEverythingWraps>
+    Hello
+  </Foo>
+
+  <Foo
+    selfClosingTags={likeThisOne}
+    stillPlaceTheBracket={onItsOwnLine}
+  />
+  ```
+
+- [`bracketSpacing`](/reference/configuration#formatterbracketspacing)
+
+  ```js title="example.js"
+  import { sort } from "sort.js";
+  const value = { sort }
+  ```
+
+  After formatting with `"bracketSpacing": false`:
+
+  ```js title="example.js"
+  import {sort} from "sort.js";
+  const value = {sort}
+  ```
+
+## VSCode extension goodies
+
+## Some CLI goodies
+
+People that rely on Biome LSP will be pleased that they can now pass a custom configuration to the command `lsp-proxy`, using the option `--config-path`. The same option is accepted by the command `start`:
+
+```shell
+biome --config-path=../path/where/config/is lsp-proxy
+biome --config-path=../path/where/config/is start
+````
+
+The CLI now exposes the option `--diagnostic-level`, that allows to filter the kind of diagnostics printed to terminal.
+
+```shell
+biome check --diagnostic-level=error ./src
+```
+
+## New lint rules, and promoted rule
+
+
+## Homage to our maintainers
+
+Since Biome was forked, new people joined the project. They have been helping with in so many ways that you can't even imagine: new features, side projects, engaging with the community, support, documentation, and more. OSS is not just about coding.
+
+Thank you to:
+- [Madeline Gurriarán @SuperchupuDev](https://github.com/SuperchupuDev)
+- [Nicolas Hedger @nhedger](https://github.com/nhedger)
+- [Victor Teles @victor-teles](https://github.com/victor-teles)
+
+## New sponsors
+
+Last but not least, we are proud to announce that we have two new sponsors"
+
+- Gold: Shiguredō (https://shiguredo.jp/)
+- Bronze: KANAME (https://www.kanamekey.com/)
+
```

Comment:
Maybe we could just say that Biome is also a linter with X stable rules with Y recommended rules?

Reply:
Let's wait for the promoted rules, then you can add something there